docs/orchestration/docker-swarm/README.md

```diff
@@ -63,8 +63,5 @@ docker stack rm minio_stack
 
 ### Explore Further
 - [Minio Erasure Code QuickStart Guide](https://docs.minio.io/docs/minio-erasure-code-quickstart-guide)
-- [Use `mc` with Minio Server](https://docs.minio.io/docs/minio-client-quickstart-guide)
-- [Use `aws-cli` with Minio Server](https://docs.minio.io/docs/aws-cli-with-minio)
-- [Use `s3cmd` with Minio Server](https://docs.minio.io/docs/s3cmd-with-minio)
-- [Use `minio-go` SDK with Minio Server](https://docs.minio.io/docs/golang-client-quickstart-guide)
-- [The Minio documentation website](https://docs.minio.io)
+- [Overview of Docker Swarm mode](https://docs.docker.com/engine/swarm/)
+- [Minio Docker Quickstart Guide](https://docs.minio.io/docs/minio-docker-quickstart-guide)
```

Comment:
Minio Docker compose link can be added here, https://docs.minio.io/docs/deploy-minio-on-docker-compose